peregrine/acquisition.py

```diff
@@ -547,6 +548,7 @@ def __init__(self, prn, carr_freq, doppler, code_phase, snr, status, signal):
     self.code_phase = code_phase
     self.status = status
     self.signal = signal
+    self.sample_index = sample_index
```

Comment:
Could you please update tracking_loop.py accordingly, where sample_index would be explicitly set to 0, when AcquisitionResult is called?